### DISCUSSION TITLE
关于自定义js、css的custom目录说明

### DISCUSSION THREAD

**Discussion by hslr-s** [hslr-s/sun-panel#128] [Announcements]
从`v1.4.0-beta24-02-20`开始，自定义目录从原来的`./web/custom`目录改成了`./custom`。

如果您是从v1.3.0版本及之前版本直接采用升级的形式更新到了`v1.4.*`，也就是保留了旧版本的配置文件和目录数据，升级不受影响。如果您是全新部署的`v1.4.*`，未保留旧版本的配置，你的自定义目录就是`./custom`，docker版本为`./conf/custom`

如果你不确定，可以参考你的配置文件[./conf/conf.ini](https://doc.sun-panel.top/zh_cn/advanced/config.html)，查看配置文件中`[base]`块下面，是否有`custom_style_path`项（如图，图中为`v1.4.0-beta24-02-20`docker版本，二进制版本应为`./custom`）。

![image](https://github.com/hslr-s/sun-panel/assets/38825747/44d045cb-b82b-4b52-99b4-8122bb18f7f8)

如果有`custom_style_path`项，值即为custom目录，之前自定义过css和js可以将`./web/custom`全部文件移动到该目录，docker版本不需要单独挂载此目录（具体参考 #98）。如果没有此配置，你的custom目录仍是（也就是v1.3.0及之前的版本）`./web/custom`

**Comment by wintsa123** [comment]
写一个feature吧，关于可以看到下载器的下载速度，不难，但不知道应该怎么分组。。。。。

**Comment by wintsa123** [comment]
提了个pr，作者有空看看
